docker-for-windows/index.md

```diff
@@ -371,38 +367,45 @@ running in experimental mode, as shown here:
 ```shell
 > docker version
 
-Client:
- Version:       18.03.0-ce
- API version:   1.37
- Go version:    go1.9.4
- Git commit:    0520e24
- Built: Wed Mar 21 23:06:28 2018
- OS/Arch:       windows/amd64
- Experimental:  true
- Orchestrator:  swarm
-
-Server:
+Client: Docker Engine - Community
```

Comment:
Align "Docker Engine - Community" with the text in that section

Reply:
This is how it is displayed in the CLI when you run the `docker version` command.